CAA record update (#3803)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/guides/custom-url-domain/main/index.md

* If you use an Okta-managed TLS certificate, you don't need a [Certificate Authorization Authority (CAA)](https://datatracker.ietf.org/doc/html/rfc6844) record. However, if you do have a CAA record, keep the following in mind:

>     * If you already have an Okta-managed TLS certificate and you later get a CAA record, Okta will be unable to renew your certificate unless you add letsencrypt.org to the issuers list, or remove the CAA record. See [Let's Encrypt - Using CAA](https://letsencrypt.org/docs/caa/).

Reviewer: I think it would be good to keep the first item.  It is still true.  These are the two ways customers can get into trouble with CAA records.  The first one will prevent them from even finishing the configuration.  The second one will prevent Okta from renewing and is a silent failure.

Author: thanks Dan - brought back the first bullet for both new and existing use cases

>     * If you already have an Okta-managed TLS certificate and you later get a CAA record, Okta will be unable to renew your certificate unless you add letsencrypt.org to the issuers list, or remove the CAA record. See [Let's Encrypt - Using CAA](https://letsencrypt.org/docs/caa/).

Reviewer 2: ```suggestion
    * If you have an Okta-managed TLS certificate and you later get a CAA record, Okta can't renew your certificate. You must either add letsencrypt.org to the issuers list or remove the CAA record. See [Let's Encrypt - Using CAA](https://letsencrypt.org/docs/caa/).
```
